src/makeFileCopy.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function getDirectoryPath(filePath) {
+  const splitedPath = filePath.split('/');
+
+  const directoryPath = splitedPath.slice(0, splitedPath.length - 1);
+
+  return directoryPath.join('/');
+}
+
+function makeFileCopy(fileToCopy, newFilePath) {
+  const fileToCopyPath = path.join(__dirname, fileToCopy);
+  const pathOfCopiedFile = path.join(__dirname, newFilePath);
+
+  if (getDirectoryPath(fileToCopyPath) === getDirectoryPath(pathOfCopiedFile)) {
+    return;
```

Comment:
It is worth informing the user that he is trying to create a copy of the file in the same directory and he needs to choose another one.